cmd/snapctl/flags.go

```diff
@@ -129,7 +125,7 @@ var (
 	flTaskMaxFailures = cli.IntFlag{
 		Name:  "max-failures",
 		Usage: "The number of consecutive failures before snap disable the task (Default 10 consective failures)",
-		Value: DefaultMaxFailures,
+		Value: -1,
```

Comment:
As an aside, @IRCody, I could pull in the default value from the scheduler package directly and use it here (which is what the code looked like before the last round of refactoring), but that wouldn't address @geauxvirtual's concerns around isolating the default value for this parameter in the scheduler package (where it belongs). In his view, no one else needs know about this parameter (and I tend to agree from a purely *separation-of-concerns* point of view), so to address that concern I decided to just pass in a `-1` in those situations where the user is interested in using the default value for this parameter (or hasn't specified a value at all) and let the scheduler pick it's own default.